schemas/2016-01-29/Microsoft.PowerBI.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-01-29/Microsoft.PowerBI.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.PowerBI",
+  "description": "Microsoft Power BI Resource Types",
+  "resourceDefinitions": {
+    "workspaceCollections": {
+      "description": "Microsoft.PowerBI/workspaceCollections resource",
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2016-01-29"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.PowerBI/workspaceCollections"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^(?:[a-zA-Z0-9]+-?)+$",
+              "minLength": 3,
+              "maxLength": 63
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "location": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-zA-Z0-9\\s]+$",
+              "minLength": 1,
+              "maxLength": 40
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "tags": {
```

Comment:
Please do not have the Azure common properties defined here as it is already defined in the root schema, reason:
1. if the pattern is updated in the root schema, it will be overwritten by RP schema
2. VS may evaluate the property improperly if the property was defined differently across RP schema and root schema.